src/quo/components/markdown/text.cljs

```diff
@@ -43,3 +43,5 @@
   [& argv]
   (let [[props children] (rn.utils/get-props-and-children argv)]
     (into [text-view props] children)))
+
+(def ?text-component [:cat [:= text] [:+ :any]])
```

Comment:
This one is here because

I can't require `markdown.text` in `schema.quo` (lint rule)
I can't require `quo.core` in `schema.quo`, seems there's a dependency loop

I guess we better have guidelines for this kind of schema?

Reply:
cc @ilmotta @mmilad75 @clauxx

Reply:
@yqrashawn, @J-Son89 not being able to put this schema in `schema.quo` is probably a good idea so we keep schemas less directly coupled with the rest of the app code.

But we can open an exception for `src/schema/quo.cljs`, wdty?

Otherwise I see other solutions:

- Remove quo schemas from `schema.quo`, thus out of the global registry. Then define reusable quo schemas within the `src/quo/` dir only.
- Relax the schema `?text-component` and not check that the first argument is an instance equal to text. Could be relaxed to simply check the first argument is hiccup. That way, you can define a general schema in the registry.

On the topic of a hiccup schema, we could just have a simplified `:schema.core/hiccup` that's non-recursive, so just checking the first level is a vector and optionally contain props and optionally children. A less protective schema, but it successfully communicates to devs that `:description-text` can be a string or component.

Btw, this `bottom-actions` component is confusing having both `description` and `description-text` props, and that `description-text` can actually be a component (not suggesting a change).

Reply:
> Relax the schema ?text-component and not check that the first argument is an instance equal to text

I prefer this one and create a general `:schema.core/hiccup`

> Btw, this bottom-actions component is confusing having both description and description-text props, and that description-text can actually be a component (not suggesting a change).

Yes, it's confusing. 
`description` should be `description-type`
`[:description {:optional true} [:maybe [:enum :top :bottom :top-error]]]`

`actions` should be `action-type`
`[:actions [:maybe [:enum :one-action :two-actions :two-vertical-actions]]]`

also, there's no variant of this `:two-vertical-actions` design in Figma, but the onboarding design did call it bottom action